### ISSUE TITLE
SyntaxError: Invalid regular expression: /(?<!node_modules.*)[\\/]node_modules[\\/](react|react-dom|scheduler|prop-types|use-subscription)[\\/]/: Invalid group in webpack-config.js

### ISSUE THREAD

**Issue by ronyperez** [zeit/next.js#9319]
Hello,
I'm working on MacOS, with NodeJS v8.9.1, npm v6..12.1.
In getting started tutorial, in setup stage, when I get to the "npm run dev" command, I get this error:

> hello-next@1.0.0 dev /Users/ronyperez/hello-next
> next

[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
**SyntaxError: Invalid regular expression: /(?<!node_modules.*)[\\/]node_modules[\\/](react|react-dom|scheduler|prop-types|use-subscription)[\\/]/: Invalid group**
    at getBaseWebpackConfig (/Users/ronyperez/hello-next/node_modules/next/dist/build/webpack-config.js:15:205)
    at HotReloader.getWebpackConfig (/Users/ronyperez/hello-next/node_modules/next/dist/server/hot-reloader.js:14:1337)
    at <anonymous>
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hello-next@1.0.0 dev: `next`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hello-next@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

When I remove (?<!node_modules.*), it works.
From tests I've done on regular expression tester online, I get this error about the asterisk:
A quantifier inside a lookbehind makes it non-fixed width

Hopes this helps

**Comment by kachkaev** [comment]
Hi @ronyperez, please see https://github.com/zeit/next.js/issues/9269

**Comment by Timer** [comment]
You're required to be running Node 8.10 or newer. Sorry about this!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
